<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AZ::SceneAPI::SceneUI::ProcessingOverlayWidget</class>
 <widget class="QWidget" name="AZ::SceneAPI::SceneUI::ProcessingOverlayWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>133</width>
    <height>123</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="leftMargin">
    <number>5</number>
   </property>
   <property name="topMargin">
    <number>5</number>
   </property>
   <property name="rightMargin">
    <number>5</number>
   </property>
   <property name="bottomMargin">
    <number>5</number>
   </property>
   <item>
    <widget class="QLabel" name="m_headerLabel">
     <property name="text">
      <string>Waiting...</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QProgressBar" name="m_progressIndicator">
     <property name="maximum">
      <number>0</number>
     </property>
     <property name="value">
      <number>-1</number>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QVBoxLayout" name="underProgressBarNotes">
     <property name="topMargin">
      <number>5</number>
     </property>
     <property name="rightMargin">
      <number>35</number>
     </property>
     <property name="bottomMargin">
      <number>8</number>
     </property>
     <item>
      <layout class="QHBoxLayout" name="subtextLayout">
       <item>
        <spacer name="subtextSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>1</width>
           <height>1</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QLabel" name="m_subtext">
         <property name="styleSheet">
          <string notr="true">
                     font-style: italic
                   </string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="m_reportArea"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
